package.json

```diff
@@ -77,13 +77,15 @@
     "react2angular": "1.1.3",
     "readable-stream": "2.3.3",
     "redux": "3.5.2",
+    "rendition": "4.6.0",
     "request": "2.81.0",
     "resin-cli-form": "1.4.1",
     "resin-cli-visuals": "1.4.1",
     "resin-corvus": "1.0.0-beta.31",
     "roboto-fontface": "0.9.0",
     "semver": "5.1.1",
     "speedometer": "1.0.0",
+    "styled-components": "https://registry.npmjs.org/styled-components/-/styled-components-3.2.6.tgz",
```

Comment:
Hmmm? Why is this a URL rather than just a version-number?

Reply:
It should be resolved by the rebase.